app/models/infra_conversion_job.rb

```diff
@@ -348,6 +348,14 @@ def wait_for_ip_address
         # If no playbook is expected to run, we don't need to wait for the IP address.
         service_template = migration_task.send("#{migration_phase}_ansible_playbook_service_template")
         if target_vm.ipaddresses.empty? && service_template.present?
+          if context["retries_#{state}".to_sym] % 10 == 0
+            target = InventoryRefresh::Target.new(
+              :association => :vms,
+              :manager_ref => {:id => target_vm.id},
```

Comment:
I don't know that `:id` is a valid manager_ref, try `:ems_ref => target_vm.ems_ref`

Reply:
I was not sure either. Changed to `ems_ref`. Thanks.